addons/mrp_subcontracting_dropshipping/tests/test_purchase_subcontracting.py

```diff
@@ -121,19 +121,13 @@ def test_mrp_subcontracting_purchase_2(self):
         po.order_line.write({'product_qty': 2})
         # Check that two deliveries with 1 component for the subcontractor have been created
```

Comment:
```suggestion
        # Check that a single delivery with the two components for the subcontractor have been created
```
Might get confusing when coming back to this test otherwise.